modules/rtp_rtcp/source/rtp_sender_video.cc

```diff
@@ -484,9 +484,9 @@ bool RTPSenderVideo::SendVideo(
   // TODO(benwright@webrtc.org) - Allocate enough to always encrypt inline.
   rtc::Buffer encrypted_video_payload;
   if (frame_encryptor_ != nullptr) {
-    if (!has_generic_descriptor) {
-      return false;
-    }
+    // if (!has_generic_descriptor) {
+    //   return false;
+    // }
```

Comment:
This prevents an Encryptor from being used if you don't have the frame header extension.  But you don't need the frame header extension, so remove this.